README

```diff
@@ -20,3 +20,32 @@ or if CPAN.pm is too old or not configured correctly, do alternatively
 
 Send suggestions, bug reports etc. to Slaven Rezic (srezic at cpan dot
 org).
+
+LICENSE AND COPYRIGHT
+
+Copyright (C) 2010-2015 Slaven Rezic (srezic at cpan dot org).
+
+
+This program is distributed under the MIT License:
```

Comment:
The actual [license of the dist](https://github.com/eserte/parse-cpan-packages-fast/blob/0.09/lib/Parse/CPAN/Packages/Fast.pm#L8) is

```perl
# This program is free software; you can redistribute it and/or
# modify it under the same terms as Perl itself.
```

This would be better accomplished by adding <https://metacpan.org/pod/Software::License::Perl_5> and/or to the `Makefile.PL` as shown here <https://perlmaven.com/how-to-add-the-license-field-to-meta-files-on-cpan>.